src/device-api/v2.ts

```diff
@@ -1,10 +1,11 @@
-import { Response, Request, Router } from 'express';
+import * as Promise from 'bluebird';
```

Comment:
bluebird should be imported as `Bluebird` to avoid confusion with the regular `Promise` that is used in other files.